src/identity.js

```diff
@@ -1286,6 +1286,7 @@ export default function Identity(mpInstance) {
      * @class mParticle.Identity.getCurrentUser().getCart()
      * @deprecated
      */
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Instead of this, you can just remove the arguments below

this applies to all areas you added this lint check